worker/stripe_charge_succeeded.js

```diff
@@ -19,39 +17,53 @@ var zeroDecimalCurrencies = [
   'XPF'
 ];
 
+var localesWith2015Email = [
+  'en-US'
+];
+
 module.exports = function(notifier_messager, mailroom) {
   const LUMBERYARD_EVENT = 'mailer';
   const FROM_EMAIL = 'The Mozilla Team <donate@mozilla.org>';
 
   return function(event, cb) {
 
-    if (event.event_type !== "stripe_charge_succeeded") {
+    if (event.event_type !== 'stripe_charge_succeeded') {
       return process.nextTick(cb);
     }
 
     var currency_code = event.data.currency.toUpperCase();
+    var locale = event.data.metadata.locale;
+
     // If this is a zero decimal currency then use it directly
     // Otherwise divide by 100 to get currency major.minor amount
-    var amount;
-    if (zeroDecimalCurrencies.indexOf(currency_code) > -1) {
-      amount = event.data.amount;
+    var amount = event.data.amount;
+    if (zeroDecimalCurrencies.indexOf(currency_code) === -1) {
+      amount = amount / 100;
+    }
+    amount = new Intl.NumberFormat(locale, {
+      style: 'currency',
+      currency: currency_code
+    }).format(amount);
+
+    var template_name = 'stripe_charge_succeeded';
+    if (localesWith2015Email.indexOf(locale) > -1) {
```

Comment:
You can also do that thing again where you default it to whatever is in the else, then just do one if. But that's just a nit.